### PR TITLE
Changleog and onload fix

### PR DESCRIPTION
**Pull request by vEpiphyte**

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -4,6 +4,33 @@
...
+
+Features and Enhancements
+-------------------------
+- Add a ``onload`` paramter to the ``stormpkg`` definition. This represents
+  a Storm query which is executed every time the ``stormpkg`` is loaded in
+  a Cortex.
+  (`#1971 <https://github.com/vertexproject/synapse/pull/1971>`_)
+- Add the ability, in Storm, to unset variables, remove items from
+  dictionaries, and remove items from lists. This is done via assigning
+  ``$lib.unset`` to the value to be removed.
```
Comment (jnwatson) [inline CHANGELOG.rst]: ```$lib.undef`??

File: CHANGELOG.rst [hunk 2]
```diff
@@ -4,6 +4,33 @@
...
+  ``$lib.undef`` to the value to be removed.
+  (`#1970 <https://github.com/vertexproject/synapse/pull/1970>`_)
+- Add support for SOCKS proxy support to the Axon and Cortex, using the
+  ``'http:proxy`` configuration option. This configuration value must be a
+  valid string for the ``aiohttp_socks.ProxyConnector.from_url()`` API. The
+  SOCKS proxy is used by the Axon when downloading files; and by the Cortex
+  when making HTTP connections inside of Storm.
+  (`#1968 <https://github.com/vertexproject/synapse/pull/1968>`_)
+- Add ``aha:admin`` to the Cell configuration to provide a common name that
+  is used to create a an admin user for remote access to the Cell via the
```
Comment (Cisphyx) [inline CHANGELOG.rst:27]: "create a an admin"

File: CHANGELOG.rst [hunk 3]
```diff
@@ -4,6 +4,33 @@
...
+-------------------------
+- Add a ``onload`` paramter to the ``stormpkg`` definition. This represents
+  a Storm query which is executed every time the ``stormpkg`` is loaded in
+  a Cortex.
+  (`#1971 <https://github.com/vertexproject/synapse/pull/1971>`_)
+- Add the ability, in Storm, to unset variables, remove items from
+  dictionaries, and remove items from lists. This is done via assigning
+  ``$lib.unset`` to the value to be removed.
+  (`#1970 <https://github.com/vertexproject/synapse/pull/1970>`_)
+- Add support for SOCKS proxy support to the Axon and Cortex, using the
```
Comment (jnwatson) [inline CHANGELOG.rst:20]: Suggest `Add support for SOCKS proxy support for outgoing connection from an Axon and Cortex```

File: CHANGELOG.rst [hunk 4]
```diff
@@ -4,6 +4,33 @@
...
+  ``'http:proxy`` configuration option. This configuration value must be a
+  valid string for the ``aiohttp_socks.ProxyConnector.from_url()`` API. The
+  SOCKS proxy is used by the Axon when downloading files; and by the Cortex
+  when making HTTP connections inside of Storm.
+  (`#1968 <https://github.com/vertexproject/synapse/pull/1968>`_)
+- Add ``aha:admin`` to the Cell configuration to provide a common name that
+  is used to create a an admin user for remote access to the Cell via the
```
Comment (jnwatson) [inline CHANGELOG.rst:27]: strike the extra 'a'

**Review by jnwatson — CHANGES_REQUESTED** [review]

**Review by invisig0th — APPROVED** [review]

File: CHANGELOG.rst [hunk 5]
```diff
@@ -4,6 +4,34 @@
 Synapse Changelog
 *****************
 
+v2.11.0 - 2020-12-30
```
Comment (invisig0th) [inline CHANGELOG.rst:7]: 2.12

**Review by invisig0th — CHANGES_REQUESTED** [review]

File: CHANGELOG.rst (hunk 5 above)
Comment (invisig0th) [reply CHANGELOG.rst:7]: and 2020-11-30 :D

**Review by invisig0th — APPROVED** [review]

**Pull request merged**